src/Util/MakerFileLinkFormatter.php

```diff
@@ -12,15 +12,16 @@
 namespace Symfony\Bundle\MakerBundle\Util;
 
 use Symfony\Component\Console\Formatter\OutputFormatterStyle;
-use Symfony\Component\HttpKernel\Debug\FileLinkFormatter;
+use Symfony\Component\ErrorHandler\ErrorRenderer\FileLinkFormatter;
+use Symfony\Component\HttpKernel\Debug\FileLinkFormatter as LegacyFileLinkFormatter;
 
 /**
  * @internal
  */
 final class MakerFileLinkFormatter
 {
     public function __construct(
-        private ?FileLinkFormatter $fileLinkFormatter = null,
+        private FileLinkFormatter|LegacyFileLinkFormatter|null $fileLinkFormatter = null,
```

Comment:
Does this fail in Symfony 6.3 where `FileLinkFormatter` (the new one) does not exist? It seems to: https://github.com/symfony/maker-bundle/actions/runs/6610525836/job/17952630835?pr=1383#step:14:74

Do we need to remove the type until 6.3 support is dropped?

Reply:
That shouldn't be the issue. I am gonna take a deeper look into the failure.

Reply:
tests fixed